logic.php

```diff
@@ -1,52 +1,89 @@
 <?php
-// Define a class to represent the form
+
+/**
+ * Defines a class to represent the form.
+ */
 class Form {
-  
-  // Public class properties
-  public $first, $last, $full, $filename, $filetemp, $phone;
-  
-  // constructor to input data
-  public function __construct($firstName, $lastName, $file_name, $file_temp, $phone) {
+
+  /** @var string $first The first name input */
+  public $first;
+  /** @var string $last The last name input */
+  public $last;
+  /** @var string $full The full name concatenated from first and last names */
+  public $full;
+  /** @var string $filename The uploaded image filename.*/
+  public $filename;
+  /** @var string $filetemp The uploaded image temporary file path.*/
+  public $filetemp;
+  /** @var int $phone The phone number input */
+  public $phone;
+
+  /**
+   * Constructor to initalize the form object
+   *
+   * @param string $firstName
+   * @param string $lastName
+   * @param string $fileName
+   * @param string $fileTemp
+   * @param int    $phone
+   * 
+   * @return void
+   */
+  public function __construct($firstName, $lastName, $fileName, $fileTemp, $phone) {
     $this->first = $firstName;
     $this->last = $lastName;
-    $this->filename = $file_name;
-    $this->filetemp = $file_temp;
+    $this->filename = $fileName;
+    $this->filetemp = $fileTemp;
     $this->phone = $phone;
   }
 
-  // Method to show the full name
+  /**
+   * Method to show the full name.
+   * 
+   * @return void
+   */
   public function showFullName() {
-    // Concatenate the first and last name and store in the $full variable
-    $this->full = $this->first . " " . $this->last;
-    
-    // Create a message that includes the full name
-    $message = "Hello " . $this->full;
+    // Concatenate the first and last name and store in the $full variable.
+    $this->full = $this->first . ' ' . $this->last;
+    // Create a message that includes the full name.
+    $message = 'Hello ' . $this->full;
 
-    // Output the message to the user
+    // Output the message to the user.
     echo $message;
```

Comment:
no need to make another varibale to echo only

Reply:
i will keep that in mind